bolt/lib/Core/ParallelUtilities.cpp

```diff
@@ -164,6 +164,15 @@ void runOnEachFunction(BinaryContext &BC, SchedulingPolicy SchedPolicy,
   Pool.wait();
 }
 
+static void EnsureAllocatorExists(BinaryContext &BC, unsigned AllocId) {
```

Comment:
```suggestion
static void ensureAllocatorExists(BinaryContext &BC, unsigned AllocId) {
```

Reply:
Better make it a closure in `runOnEachFunctionWithUniqueAllocId()` unless you plan to use it more in follow-up patches.

Reply:
Ah, yes, it's better as a closure.
I've now moved it to a closure, see the latest commit.